apis/core/bootstrap/shapes/dataset.ts

```diff
@@ -0,0 +1,90 @@
+import { cc, shape } from '@cube-creator/core/namespace'
+import { hydra, rdfs, sh, dcat, dcterms, schema } from '@tpluscode/rdf-ns-builders'
+import { turtle } from '@tpluscode/rdf-string'
+
+export const DatasetShape = turtle`
+${shape('dataset/edit-metadata')} {
+  ${shape('dataset/edit-metadata')} a ${sh.NodeShape}, ${hydra.Resource} ;
+    ${sh.targetClass} ${dcat.DataSet} ;
+    ${rdfs.label} "Cube Metadata" ;
+    ${sh.property} [
+      ${sh.name} "Title" ;
+      ${sh.path} ${dcterms.title} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.minLength} 1 ;
+      ${sh.order} 10 ;
+    ] ;
+    ${sh.property} [
+      ${sh.name} "Description" ;
+      ${sh.path} ${dcterms.comment} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.minLength} 1 ;
+      ${sh.order} 20 ;
+    ] ;
+    ${sh.property} [
+      ${sh.name} "Version" ;
+      ${sh.path} ${schema.version} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.minLength} 1 ;
+      ${sh.order} 30 ;
+    ] ;
```

Comment:
This one too, I suppose it would be set behind the scenes and only when we actually publish a cube. Might not even have it as part of the metadata at all